templates/homepage.html

```diff
@@ -16,7 +16,7 @@ <h1>Web &amp; design team portal</h1>
         <p class="p-card__content">Get the site status deployements</p>
       </div>
       <div class="col-4 p-card--highlighted">
-        <a href="https://canonical-web-and-design.github.io/practices/"> <h3 class="p-card__title">Practices</h3></a>
+        <a href="https://discourse.canonical.com/c/web-and-design-team/coding"> <h3 class="p-card__title">Practices</h3></a>
```

Comment:
Those practices were for the entire team (project management, design and devs). I am not sure this is the replacement of this site.

@nottrobin  @anthonydillon  are the old practices hosted somewhere new now?

Reply:
This is right, all guides, practise and team documentation is being centralised on the Canonical discourse.

Reply:
Well, at the moment https://discourse.canonical.com/c/web-and-design-team/7 contains all practices, but also all guides. You could point there but I think it might make sense to change the "Practices" text to be more like "Knowledge base" or something.

We might want to think about if there's a way to structure the practices so they can actually all exist within a dedicated practices space, rather than appearing all mixed in with guides like they do at the moment.

Reply:
@pmahnke (I think) pointed out that these guides are also internal.

So maybe what we should do is change the link text to "Knowledge base (internal)" and link to https://discourse.canonical.com/c/web-and-design-team/7 for now. But down the line we should definitely reimplement the practices site here.

Filed https://github.com/canonical-web-and-design/webteam.canonical.com/issues/91